packages/core-mobile/app/new/features/bridge/screens/BridgeScreen.tsx

```diff
@@ -630,15 +649,22 @@ export const BridgeScreen = (): JSX.Element => {
     sourceNetwork
   ])
 
+  usePreventScreenRemoval(isPending)
```

Comment:
a bridge can take hours to complete so I don't think we should prevent users from closing this screen.